src/Microsoft.Azure.WebJobs.Host/Executors/IStorageAccountProvider.cs

```diff
@@ -9,6 +9,6 @@ namespace Microsoft.Azure.WebJobs.Host.Executors
 {
     internal interface IStorageAccountProvider
     {
-        Task<IStorageAccount> GetAccountAsync(string connectionStringName, CancellationToken cancellationToken);
+        Task<IStorageAccount> TryGetAccountAsync(string connectionStringName, CancellationToken cancellationToken);
```

Comment:
Please rename this to GetAccountOrNullAsync - TryGet by convention is a method that returns a boolean value and returns its value as an out parameter.